packages/nx-stylelint/src/executors/lint/schema.json

```diff
@@ -45,6 +45,11 @@
       "type": "boolean",
       "description": "Hide output text.",
       "default": false
+    },
+    "allowEmptyInput": {
+      "type": "boolean",
+      "description": "Allow no results from the file patterns to exit without an error.",
+      "default": false
```

Comment:
Please set the default to true

Reply:
will do, true is definitely better IMO but wasn't sure what to do